contributing.md

```diff
@@ -21,8 +21,8 @@ The plugin will be served from [http://localhost:3000/](http://localhost:3000/).
 Before publishing this plugin, make sure:
 
 * you've properly described any configuration parameters in this README file;
-* you've properly compiled this project's `package.json` following the [official rules](https://www.datocms.com/docs/plugins/publishing/);
-* you've added a cover image (`cover.png`) and a preview GIF (`preview.gif`) into the `docs` folder.
+* you've updated the changelog with relevant information;
+* you've properly compiled this project's `package.json` following the [official rules](https://www.datocms.com/docs/plugins/publishing/).
```

Comment:
The official rules link redirects to a 'legacy plugins' section, is this plugin in an older format?

Reply:
It was once in an older format. So this should have the links to the new plugin section

Reply:
https://www.datocms.com/docs/plugin-sdk/publishing-to-marketplace

Reply:
cb8b4cf

Reply:
cb8b4cf